doc/source/faq.rst

```diff
@@ -110,78 +110,6 @@ details.
 Visualizing Data in Qt applications
```

Comment:
Yes. The second part means ecosystem.rst which lists some related projects.

- https://github.com/pydata/pandas/blob/master/doc/source/ecosystem.rst